sensor_array/analysis/execute_genetic_algorithm_analysis.py

```diff
@@ -29,31 +29,42 @@
     mof_densities.copy()
     mof_densities.update({ mof : data['mofs'][mof]['density']})
 
+# Import results as dictionary
+sim_results_import = read_data_as_dict(sim_data)
+exp_results_import = read_data_as_dict(exp_data)
+
 # --------------------------------------------------
 # ----- Calculate all single MOF PMFs --------------
 # --------------------------------------------------
-(exp_results_full, exp_results_mass, exp_mof_list) = (
-    import_experimental_data(exp_results_import, mof_list, mof_densities, gases) )
-(sim_results_full) = (
-    import_simulated_data(sim_results_import, mof_list, mof_densities, gases) )
-(element_pmf_results) = (
-    calculate_element_pmf(exp_results_full, sim_results_full, mof_list, stdev, mrange) )
+exp_results_full, exp_results_mass, exp_mof_list = \
+    import_experimental_data(exp_results_import, mof_list, mof_densities, gases)
+sim_results_full = \
+    import_simulated_data(sim_results_import, mof_list, mof_densities, gases)
+comp_list, mole_fractions = \
+    create_comp_list(sim_results_full, mof_list, gases)
+sim_results_full = \
+    moving_average_smooth(sim_results_full, mof_list, gases, comp_list, mole_fractions, num_points = 2)
+sim_results_full = \
+    reintroduce_random_error(sim_results_full, error=1, seed=0)
+exp_results_full = \
+    convert_experimental_data(exp_results_full, sim_results_full, mof_list, gases)
+element_pmf_results = \
+    calculate_element_pmf(exp_results_full, sim_results_full, mof_list, stdev, mrange)
 
 # --------------------------------------------------
 # ----- Run the Genetic Algorithm ------------------
 # --------------------------------------------------
 
-array_size_vect = [10, 15, 20, 25, 30, 35, 40, 45]
-# array_size_vect = [1,2,3,4,5]
-num_runs = 3
+array_size_vect = [5, 10, 15, 20, 25, 30, 35, 40, 45]
+num_runs = 1
```

Comment:
commit message is vague and doesn't indicate these changes to the runs / array_size_vect